File: 1929-concatenation-of-array/1929-concatenation-of-array.py
class Solution(object):
    def getConcatenation(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        
        #1. Determine the length of nums array 
        ans = []
        length = len(nums)
        
        #2. Append stuff
        for num in nums:
            ans.append(num)
        
        for num in nums:
            ans.append(num)
        
        
        return ans 
        
        
            
        

File: 1929-concatenation-of-array/1929-concatenation-of-array.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int[] ans = new int[nums.length*2];
        int counter = 0;
        for(int i=0;i<nums.length;i++){
            ans[counter] = nums[i];
            counter++;
        }
        
        for(int i=0;i<nums.length;i++){
            ans[counter] = nums[i];
            counter++;
        }
        
        return ans;
        
    }
}